lua/wikis/valorant/Infobox/Person/Player/Custom.lua

```diff
@@ -188,13 +189,16 @@ function CustomPlayer:getPersonType(args)
 	return {store = 'player', category = 'Player'}
 end
 
----@return string?
+---@return Widget?
 function CustomPlayer:createBottomContent()
 	if self:shouldStoreData(self.args) and String.isNotEmpty(self.args.team) then
 		local teamPage = Team.page(mw.getCurrentFrame(), self.args.team)
-		return
-			tostring(MatchTicker.player{recentLimit = 3}) ..
-			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing tournaments of', {team = teamPage})
+		return HtmlWidgets.Fragment{
+			children = {
+				MatchTicker.player{recentLimit = 3},
+				UpcomingTournaments{name = teamPage, onlyHighlightOnValue = 'highlighted'}
```

Comment:
onlyHighlightOnValue is redundant with how the highlightCondition looks for this wiki

https://github.com/Liquipedia/Lua-Modules/blob/main/lua/wikis/valorant/HighlightConditions.lua